src/auxil/auxil/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gfx-auxil"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
I think spirv_cross is a public dependency of this crate, so it has to jump to 0.4.0

Reply:
Good catch, thanks!